src/Events/Clients/EventsQueueClient.cs

```diff
@@ -2,22 +2,19 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Text;
 using System.Threading.Tasks;
-
+using Altinn.Platform.Events.Clients.Interfaces;
 using Altinn.Platform.Events.Configuration;
 using Altinn.Platform.Events.Models;
-using Altinn.Platform.Events.Services.Interfaces;
-
 using Azure.Storage.Queues;
-
 using Microsoft.Extensions.Options;
 
-namespace Altinn.Platform.Events.Services
+namespace Altinn.Platform.Events.Clients
 {
     /// <summary>
     /// The queue service that handles actions related to the queue storage.
```

Comment:
The summary still use the term service. Be as specific as you dare to be. Something like:
```suggestion
    /// Implementation of the <see ref="IEventsQueueClient"/> using Azure Storage Queues.
```